app/src/mainwindow2.cpp

```diff
@@ -121,6 +122,9 @@ MainWindow2::MainWindow2(QWidget* parent) :
 
     readSettings();
 
+    mZoomLabel = new QLabel("");
+    ui->statusbar->addWidget(mZoomLabel);
+
```

Comment:
I could remove these lines, but then I must remove
- the cal to updateZoomLabel() in line 128
- the connect with viewChanged and updateZoomLabel, in line 133
- the function updateZoomLabel() in line 1523
- the declaration of updateZoomLabel(), in line 127 in mainwindow2.h
Then there will be no zoom label at all.
Should I do it?

Reply:
What I meant was essentially to restore the state of the status bar from before this PR. But if it is too much trouble, feel free to leave it as is and I will simply rebase my status bar PR as necessary.

Reply:
(Resubmitting this comment because of some GitHub weirdness)

What I meant was essentially to restore the state of the status bar from before this PR. But if it is too much trouble, feel free to leave it as is and I will simply rebase my status bar PR as necessary.